pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -240,8 +291,8 @@ func (r *ReconcileUserSignup) provisionMasterUserRecord(userSignup *toolchainv1a
 			"Error creating MasterUserRecord")
```

Comment:
don't forget to change the lines [285-289](https://github.com/codeready-toolchain/host-operator/pull/93/files#diff-5b4b18a0fc2e0d708e4e6334cdb6bc69R285-R289) - if the MUR already exists, then I guess that we should change (increment) the suffix of the compliant username

Reply:
I've refactored this logic now.